scripts/tasks.sh

```diff
@@ -74,10 +74,12 @@ function tasks() {
         rook-10-to-14-images|rook_10_to_14_images)
             rook_upgrade_tasks_load_images "from-version=1.0" "to-version=1.4"
             ;;
-        rook-upgrade)
+        rook-upgrade|rook_upgrade)
```

Comment:
that ringed the bell but after look the implementation I saw that we are doing in this away for all cases, so OK,